docs/specifications/governance.md

```diff
@@ -0,0 +1,66 @@
+- **Event phase** - the 
+- **Governance** - rules and processes that are used for making decisions concerning permission-less applications and platforms
+- **Participation** - interacting with or taking part in an event, either by voting or staking funds
+- **Proposal** - synonymous with voting event, which is an event that contains one or more questions to be voted on by users
+- **Vote** - participating in a voting event, initially by selecting answers (aka votes) for each question on the proposal's ballot then broadcasting a transaction with appropriately structured metadata (so that it can be tracked by the node(s))
+- **Voting output** - a designated output to be used for all voting operations (e.g. increasing or decreasing voting power and voting or unvoting for an event) 
+- **Voting power** - the amount of votes a user can cast for a proposal, which is based off of how much IOTA or SMR he or she has; this **MAY** or **MAY NOT** be the same as a user's total balance, it depends on how much they have manually designated as their voting power
+
+## Adding a Proposal
+
+To be able to vote on a proposal, a user first needs to add it to his or her dashboard. This can happen in multiple ways:
+
+- Automatically registering the proposals that are tracked by the client options node(s)
+- Manually entering a proposal (aka event) ID and a URL of a node that is tracking it
+- Following a deep link containing a proposal (aka event) ID and a URL of a node that is tracking it
+
+Once added, Firefly will regularly poll the given node(s) to update the states of the proposals, which includes the current vote results. 
+
+Users may also remove proposals from their account(s), but only if they are **NOT** currently voting for the proposal.
+
+## Voting on a Proposal
+
+### Managing Voting Power
+
+Users must have a non-zero voting power to be able to vote for a proposal. To do this, they simply click the "Manage voting power" button from the Governance dashboard.
+
+Funds are designated for voting by creating a voting output, i.e. a basic output that is tagged with the `PARTICIPATE` keyword. A user can choose to designate any non-zero amount less than or equal to their available balance to become his or her voting power.
+
+Voting outputs are inconsumable for all transactions, unless the user is...
+
+- Increasing or decreasing voting power
+- Voting or unvoting for a proposal
+
+By consequence, this means that voting power is **NOT** considered a part of a user's available balance.
+
+### Casting a Vote
+
+In order to cast a vote for a proposal, there are two steps:
+
+1. Select an answer for every question on the proposal's ballot
+2. Broadcast a transaction containing those votes as both a tagged data payload (what is tracked by the nodes) and output metadata
+
+If this is the first proposal a user is voting on, the data payload and output metadata will be added alongside the existing `PARTICIPATE` tag, otherwise
```

Comment:
```suggestion
If the user votes for the first time, the data payload and output metadata will be added alongside the existing `PARTICIPATE` tag, otherwise
```